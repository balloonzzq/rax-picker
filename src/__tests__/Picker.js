import {createElement, Component} from 'rax';
import renderer from 'rax-test-renderer';
import Picker from '..';

describe('Picker', () => {
  it('should render Picker', () => {
    const tree = renderer.create(
      <Picker>
        <Picker.Item value={'man'} label={'man'} />
      </Picker>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
