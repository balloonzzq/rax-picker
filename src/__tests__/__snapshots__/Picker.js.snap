// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Picker should render Picker 1`] = `
Object {
  "children": Array [
    Object {
      "attributes": Object {
        "selected": "selected",
        "value": "man",
      },
      "children": Array [
        "man",
      ],
      "tagName": "OPTION",
    },
  ],
  "eventListeners": Object {
    "change": [Function],
  },
  "style": Object {
    "display": "flex",
    "flexDirection": "row",
    "justifyContent": "center",
  },
  "tagName": "SELECT",
}
`;
